src/backend/controllers/casusController.js

```diff
@@ -1,23 +1,22 @@
 // @flow strict
 
-const User = require('../../models/userModel');
+// NOTE: This functionality is not implemented in v1.0
```

Comment:
Someday you will be formally declared a saint, Emil, for all your helpful documentation to future groups.